src/components/card/_card--basic-menu.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "@card--basic" %}
+
+{% block card_header %}
+<header class="usa-card__header">
+  <h2 class="usa-card__heading">
+    {{ card.title }}
+  </h2>
+</header>
+{% endblock %}
+
+{% block card_body %}
+<p>
+  A card is distinct from its collection peers in content.
+</p>
+{% endblock %}
+
+{%- block card_footer -%}
+  <ul class="add-list-reset">
+    <li class="display-inline-flex margin-right-05">
+      <a class="text-no-underline text-base-light" href="#" aria-label="Placeholder icon link">
+        <svg style="width:24px;height:24px" viewBox="0 0 24 24">
+          <path fill="currentColor" d="M1,21H23L12,2" />
+        </svg>
+      </a>
+    </li>
+    <li class="display-inline-flex margin-right-05">
+      <a class="text-no-underline text-base-light" href="#" aria-label="Placeholder icon link">
+        <svg style="width:24px;height:24px" viewBox="0 0 24 24">
+          <path fill="currentColor" d="M3,3V21H21V3" />
+        </svg>
+      </a>
+    </li>
+    <li class="display-inline-flex">
+      <a class="text-no-underline text-base-light" href="#" aria-label="Placeholder icon link">
+        <svg style="width:24px;height:24px" viewBox="0 0 24 24">
+          <path fill="currentColor" d="M12,2.5L2,9.8L5.8,21.5H18.2L22,9.8L12,2.5Z" />
+        </svg>
+      </a>
+    </li>
+  </ul>
+{%- endblock %}
```

Comment:
It's used in the individual examples in Fractal. 

This doesn't show up on the component page because the combined example seemed like a better layout for doc purposes.

Reply:
I might consider removing this one as a variant as the functionality is kinda obscure